src/Common/Models/SliceModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Middleware.Common.Models
+{
+    internal class SliceModel
+    {
+        public Guid SliceId { get; set; }
+        public string SliceName { get; set; }
+        public bool DynamimcSlcie { get; set; }
+        public  string TypeSlice { get; set; }
+    }
```

Comment:
Same as before. I would suggest to have following property names:

* Id
* Name
* IsDynamic
* Type
We know the context :)

Reply:
done